bip-0001.mediawiki

```diff
@@ -67,7 +67,7 @@ Each BIP should have the following parts:
 
 * Abstract -- a short (~200 word) description of the technical issue being addressed.
 
-* Copyright/public domain -- Each BIP must either be explicitly labelled as placed in the public domain (see this BIP as an example) or licensed under the Open Publication License.
+* Copyright/public domain -- Each BIP must either be explicitly labelled as placed in the public domain (see this BIP as an example).
```

Comment:
This sentence no longer makes sense.

I'd suggest the following:

```mediawiki
* Copyright -- Each BIP must be explicitly labeled as placed in the public domain or licensed under at least one of the following licenses:
** BSD-2-Clause: [https://opensource.org/licenses/BSD-2-Clause OSI-approved BSD 2-clause license]
** CC0-1.0: [https://creativecommons.org/publicdomain/zero/1.0/ Creative Commons CC0 1.0 Universal]
** CC-BY-4.0: [https://creativecommons.org/licenses/by/4.0/ Creative Commons Attribution 4.0 International]
```

This would allow us to keep all the BIPs that are currently not placed in the public domain instead of having to delete them.